### PR TITLE
Configure overcommit

### PR DESCRIPTION
**Pull request by zhinfuhnya**
https://www.pivotaltracker.com/n/projects/2167210/stories/178965253

### CODE REVIEW

File: .overcommit.yml [hunk 1]
```diff
@@ -37,6 +27,13 @@
 # other checks that you want to run before you allow a commit object to be
 # created.
 PreCommit:
+  RuboCop:
+    enabled: true
+    on_warn: fail
+  TrailingWhitespace:
+    enabled: true
+    exclude:
+      - '**/db/structure.sql'
```
Comment (Riddlerrr) [inline .overcommit.yml:36]: I think we don't have structure.sql, we use `schema.rb` instead, so it can be deleted

**Review by Riddlerrr — APPROVED** [review]

**Comment by Riddlerrr** [comment]
I've also moved rubocop config to default path to project root directory

**Review by Alexey1100 — CHANGES_REQUESTED** [review]
Interesting, lots of files got autocorrected while config hasn't changed.
Is it working properly?

**Comment by zhinfuhnya** [comment]
> Interesting, lots of files got autocorrected while config hasn't changed.
> Is it working properly?

@Alexey1100 in my case RuboCop works without autocorrection, it just points to the files that need to be corrected.

**Comment by Alexey1100** [comment]
@zhinfuhnya we have changes checked in here which shouldn't be here, because the config hasn't changed.

**Comment by zhinfuhnya** [comment]
@Alexey1100 Indeed, if I make a change for one line, RuboCop check on the whole file.

**Comment by Riddlerrr** [comment]
I've updated rubocop_todo file so now it works good.
I've also enabled more checks.

**Comment by Alexey1100** [comment]
Seems like what I'm seeing is related to:

https://github.com/sds/overcommit/issues/704

**Comment by Alexey1100** [comment]
Fix above is not yet released, so resolved by manually adding `--disable-pending-cops` flag to overcommit rubocop config.

**Review by Alexey1100 — APPROVED** [review]

**Pull request merged**